src/GeneralStateTestsFiller/stEWASMTests/utils/callStaticFiller.yml

```diff
@@ -0,0 +1,90 @@
+# Basic EWASM interface test for callStatic
+---
+callStatic:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: '0'
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: |
+        (module
+        (import "ethereum" "callStatic" (func $callStatic (param i64 i32 i32 i32) (result i32)))
+        ;;(import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+        (import "debug"    "printMemHex"  (func $printMemHex (param i32 i32)))
+        (memory 1)
+        ;; 0-20: Address
+        ;; 
+        (data (i32.const 0) "\0b\bf\6e\7e\67\31\53\c1\97\86\2a\8e\bc\ed\e5\fc\74\53\4f\c9")
+        (data (i32.const 20) "\12\34")
+        (export "main" (func $main))
+        (export "memory" (memory 0))
+        (func $main
+          (i32.store
+            (i32.const 54)
+            (call $callStatic
+              (i64.const 200000)
+              (i32.const 0)
+              (i32.const 20)
+              (i32.const 2)))
+          (call $printMemHex (i32.const 54) (i32.const 32))
+          ))
+      nonce: ''
+      storage: {}
+    c94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: |
+        (module
+        (import "ethereum" "getAddress" (func $getAddress (param i32)))
+        (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+        (import "debug"    "printMemHex" (func $printMemHex (param i32 i32)))
+        (memory 1)
+        (export "main" (func $main))
+        (export "memory" (memory 0))
+        (func $main
+        (call $getAddress (i32.const 32))
+        (call $storageStore (i32.const 98) (i32.const 0))
```

Comment:
This is `storage[0] = 0x0`, perhaps store a value other than zero here and check that the static call was correct by making sure the storage location remains 0.

Reply:
I was trying to store the address I got in previous step, but the offset is wrong, must be 32 instead of 0, I'll fix it.